build/cmake/lib/CMakeLists.txt

```diff
@@ -143,10 +143,18 @@ IF (ZSTD_BUILD_STATIC)
             OUTPUT_NAME ${STATIC_LIBRARY_BASE_NAME})
 ENDIF (ZSTD_BUILD_STATIC)
 
+
+GET_PROPERTY( LIB64 GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS )
```

Comment:
Apparently, `FIND_LIBRARY_USE_LIB64_PATHS ` requires cmake >= v3.0.2.
https://cmake.org/cmake/help/v3.0/prop_gbl/FIND_LIBRARY_USE_LIB64_PATHS.html

Note that `CMakeLists.txt` is currently compatible with >= v2.8.9 : 
https://github.com/facebook/zstd/blob/dev/build/cmake/CMakeLists.txt#L11
As a general rule, increasing the minimum version number is not desirable, as it correspondingly decreases the number of distributions which can satisfy the dependency.

We currently try to keep `cmake` script compatible with versions <= 2.8.12, as it is the version shipped within Ubuntu Trusty, an actively supported LTS release, used by several CI systems such as TravisCI and CircleCI.

Reply:
This is the one I was referring to https://cmake.org/cmake/help/v2.8.12/cmake.html v2.8.12, and Probably  my mistake I should have approached for make file as well

Reply:
The intention was to let `cmake` or `make` decide install location based on arch, I believe the `cmake` property `FIND_LIBRARY_USE_LIB64_PATHS` should help in this. If you feel this is worth an effort, specially considering there will be a change in install location for future releases. I'll go ahead and do a bit of work on `Makefile` as well.